registry/consul/routecmd.go

```diff
@@ -26,6 +26,9 @@ type routecmd struct {
 func (r routecmd) build() []string {
 	var svctags, routetags []string
 	for _, t := range r.svc.ServiceTags {
+
+		t = strings.TrimSpace(t)
```

Comment:
@logocomune does this trim all spaces, or only leading/trailing spaces?

Reply:
> TrimSpace returns a slice of the string s, with all leading and trailing white space removed, as defined by Unicode.

https://pkg.go.dev/strings#TrimSpace